pom.xml

```diff
@@ -525,6 +545,17 @@ under the License.
       </build>
     </profile>
     <!-- END SNIPPET: release-profile -->
+    <profile>
+      <id>jdk9ornewer</id>
```

Comment:
Make is simple: `jdk9+`?

Reply:
Done in https://github.com/apache/maven-apache-parent/pull/188/commits/8ab37cef72208e2b9667dbd298a446b16696ec66.